.changeset/olive-cooks-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphiql/react': patch
+---
+
+Mark `graphql` as external dependency to avoid importing multiple instances
```

Comment:
i like to see that you are trying to provide separate changelog entries like this to provide a clean changelog, but in the future, for multiple changes in the same commit you can just use a markdown list in the same changeset file!